src/index.ts

```diff
@@ -482,6 +537,20 @@ export class UnleashClient extends TinyEmitter {
                         type: 'HttpError',
                         code: response.status,
                     });
+                } else if (response.status !== 304) { // Not an error (like 2xx)
+                    this.etag = response.headers.get('ETag') || '';
+                    const data = await response.json();
+                    await this.storeToggles(data.toggles);
+
+                    if (this.sdkState !== 'healthy') {
+                        this.sdkState = 'healthy';
+                    }
+
+                    this.emitReady();
+                    
+                    this.storeLastRefreshTimestamp();
+                } else if (response.status === 304)  { // Specific 304
```

Comment:
else suffirait non ?
```suggestion
                } else { // Specific 304
```

Reply:
Sinon ça serait peut-être plus simple/lisible de modifier le code du `!response.ok` en ajoutant un `if` à l'intérieur ?

```ts
if (!response.ok) {
    if(response.status === 304) {
      this.storeLastRefreshTimestamp();
   } else {
   ...
}
```

Reply:
oui tu as raison j'ai meme mis des commentaires tellement c'était pas clair :) .. je fais la modif